toc.hbs.md

```diff
@@ -535,6 +535,7 @@
       - [Spring Boot conventions overview](spring-boot-conventions/about.hbs.md)
       - [Install Spring Boot conventions](spring-boot-conventions/install-spring-boot-conventions.hbs.md)
       - [Configure Spring Boot actuators](spring-boot-conventions/configuring-spring-boot-actuators.hbs.md)
+      - [Configuring Liveliness Readiness Startup Probes](spring-boot-conventions/configuring-liveliness-readiness-startup-probes.hbs.md)
```

Comment:
Maybe it is worth to add alpha in the TOC as well: 
`      - [Configuring Liveliness Readiness Startup Probes (alpha)](spring-boot-conventions/configuring-liveliness-readiness-startup-probes.hbs.md)`